routes/api-router.ts

```diff
@@ -1,18 +1,19 @@
-import {endpointsRouter} from "./endpoints-router";
+import { endpointsRouter } from "./endpoints-router";
 
 const apiRouter = require("express").Router();
-// const endpointsRouter = require("./endpoints-router");
-const {katasRouter} = require("./katas-router");
-const {usersRouter} = require("./users-router");
-const {topicsRouter} = require("./topics-router");
+
+const { katasRouter } = require("./katas-router");
+const { usersRouter } = require("./users-router");
+const { topicsRouter } = require("./topics-router");
+const { testRouter } = require("./kata-test-router");
 //Require in routers the come off of this point
 
 apiRouter.use("/", endpointsRouter);
 apiRouter.use("/katas", katasRouter);
 apiRouter.use("/users", usersRouter);
 apiRouter.use("/topics", topicsRouter);
+apiRouter.use("/test", testRouter);
 /* Adv
 apiRouter.use("/solutions", solutionsRouter)
-apiRouter.use("/test", testRouter) 
 */
-module.exports = apiRouter
\ No newline at end of file
+export default apiRouter;
```

Comment:
Why `export default` instead of `module.exports`? Isn't `export default` React syntax?